src/main/java/codeu/model/data/User.java

```diff
@@ -89,4 +91,19 @@ public void setAdmin(Boolean admin){
     this.admin = admin;
   }
 
+  /** Returns the Instant into a String time format to display to users. */
+  public String getTime() {
+    LocalDateTime localDate = LocalDateTime.ofInstant(creation, ZoneId.systemDefault());
+    int hour = localDate.getHour();
+    String timeAMPM = "";
+    if (hour > 12) {
```

Comment:
Come on.. with all the time and date stuff they've added to java in the last couple of versions, nobody does date formatting like this! Surely there's a better way.. have you tried eg SimpleDateFormat ?

If this were a more serious app of course you'd want adjust the timezone as well as the format to suit the user but we can skip that here. (Here, we do everything in "Google Standard Time")

Reply:
hahahah whoops! I actually didn't implemented the method I just copied and pasted it from codeu.model.data.message for a quick fix 😆